zap/src/main/java/org/zaproxy/zap/extension/ascan/PopupMenuActiveScanCustomWithContext.java

```diff
@@ -50,4 +53,15 @@ public PopupMenuActiveScanCustomWithContext(ExtensionActiveScan extension) {
                     extension.showCustomScanDialog(new Target(context));
                 });
     }
+
+    @Override
+    public boolean isEnableForComponent(Component invoker) {
+        if (invoker instanceof JTree
+                && SiteMapPanel.CONTEXT_TREE_COMPONENT_NAME.equals(invoker.getName())) {
+            JTree contextTree = (JTree) invoker;
+
+            this.setEnabled(contextTree.getSelectionCount() < 2);
```

Comment:
Could this be done directly without the intermediate assignment? Something like:
```java
this.setEnabled((JTree) invoker.getSelectionCount() < 2);
```